contracts/src/v0.8/dev/interfaces/IVRFV2PlusMigration.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IVRFV2PlusMigration {
```

Comment:
Lets insert some doc comments here. And I just noticed that we already have a `IVRFCoordinatorV2PlusMigration`, apologies for the bad naming recommendation. Maybe just `IVRFV2PlusMigrate`?

```suggestion
/// @notice This interface is implemented by all VRF V2+ coordinators that can
/// @notice migrate subscription data to new coordinators.
interface IVRFV2PlusMigrate {
```

Reply:
Sure, fixed this